data/changelog/ocaml/2024-04-16-ocaml-5.2.0-beta2.md

```diff
@@ -0,0 +1,93 @@
+---
+title: OCaml 5.2.0 - Second Beta
+description: Second Beta Release of OCaml 5.2.0
+tags: [ocaml]
+changelog: |
+  ### Compiler-libs API Changes
+  
+  - [#13001](https://github.com/ocaml/ocaml/issues/13001): do not read_back entire shapes to get aliases' uids when building the
+    usages index
+    (Ulysse Gérard, review by Gabriel Scherer and Nathanaëlle Courant)
+  
+  ### Bug Fixes
+  
+  - [#13058](https://github.com/ocaml/ocaml/issues/13058): Add TSan instrumentation to caml_call_gc(), since it may raise
+    exceptions.
+    (Fabrice Buoro, Olivier Nicole, Gabriel Scherer and Miod Vallat)
+  
+  - [#13079](https://github.com/ocaml/ocaml/issues/13079): Save and restore frame pointer across Iextcall on ARM64
+    (Tim McGilchrist, review by KC Sivaramakrishnan and Miod Vallat)
+  
+  - [#13094](https://github.com/ocaml/ocaml/issues/13094): Fix undefined behavior of left-shifting a negative number.
+    (Antonin Décimo, review by Miod Vallat and Nicolás Ojeda Bär)
+  
+  ### Documentation Updates
+  
+  - [#13078](https://github.com/ocaml/ocaml/issues/13078): update Format tutorial on structural boxes to mention alignment
+    questions.
+    (Edwin Török, review by Florian Angeletti)
+  
+  - [#13092](https://github.com/ocaml/ocaml/issues/13092): document the existence of the `[@@poll error]` built-in attribute
+    (Florian Angeletti, review by Gabriel Scherer)
+
+  - [#13066](https://github.com/ocaml/ocaml/issues/13066), update OCAMLRUNPARAM documentation for the stack size parameter l
+    (Florian Angeletti, review by Nicolás Ojeda Bär, Tim McGilchrist, and
+     Miod Vallat)
+ ---
+
+Last week, we merged an update to the compiler-libs "shape" API for querying definition information from the compiler.
+
+Unfortunately, this small change of API breaks compatibility with at least odoc.
+Generally, we try to avoid this kind of changes during the beta releases of the
+compiler. However, after discussions we concluded that it will be easier on the
+long term to fix the API right now in order to avoid multiplying the number of
+supported versions of the shape API in the various OCaml developer tools .
+
+We have thus released a second beta version of OCaml 5.2.0 to give the time to developer tools to update their 5.2.0 version ahead of the release (see below for the installation instructions).
```

Comment:
```suggestion
We have thus released a second beta version of OCaml 5.2.0, so there will be time for the developer tools to update their 5.2.0 version ahead of the release (see below for the installation instructions).
```

I'm a little confused by this sentence. Does the above suggestion capture what you mean?